### PR TITLE
Localization for Pages

### PR DESCRIPTION
**Pull request by davidwessman**
Adding translation to Page title and PageElement header + text

### CODE REVIEW

File: db/migrate/20160629204225_create_translation_for_pages.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateTranslationForPages < ActiveRecord::Migration
+  def up
+    Page.create_translation_table!({ title: { type: :string, null: :false, default: '' } },
+                                   { migrate_data: true })
```
Comment (davidwessman) [inline db/migrate/20160629204225_create_translation_for_pages.rb]: These follow the documentation of Globalize

File: db/migrate/20160629204225_create_translation_for_pages.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    PageElement.create_translation_table!({ text: :text, headline: :string }, { migrate_data: true })
```
Comment (davidwessman) [inline db/migrate/20160629204225_create_translation_for_pages.rb]: These follow the documentation of Globalize

File: app/models/page.rb [hunk 3]
```diff
@@ -6,14 +6,18 @@ class Page < ActiveRecord::Base
   has_many :page_images, dependent: :destroy
 
   # Validations
-  validates :title, presence: true
+  validates :title, :title_sv, :title_en, presence: true
```
Comment (navro) [inline app/models/page.rb]: I think `_en` attributes should be optional

**Comment by navro** [comment]
+1

**Pull request merged**